Makefile

```diff
@@ -17,8 +17,6 @@ help:
 	@echo "make test              Run the unit tests"
 	@echo "make functests         Run the functional tests"
 	@echo "make bddtests          Run the gherkin tests"
-	@echo "make docstrings        View all the docstrings locally as HTML"
```

Comment:
Why has this gone away? I haven't used this, but it doesn't seem directly related.